### PR TITLE
Add freedreno support to mesa and libdrm

### PR DESCRIPTION
**Pull request by opendata26**
I have tested these changes with my z2 tablet so they should work fine for other adreno devices.

### CODE REVIEW

**Comment by opendata26** [comment]
@ncopa  done

**Comment by ncopa** [comment]
can you squash it into the previous commit and `git push --force`?

**Comment by jirutka** [comment]
This PR contains changes in two separate abuilds, so there should be two separate commits (with log title in format `<repo>/<pkgname>: …`), one for each abuild.

**Comment by opendata26** [comment]
Done

**Comment by opendata26** [comment]
@jirutka look ok?

File: main/libdrm/APKBUILD [hunk 1]
```diff
@@ -23,6 +23,7 @@ build() {
 		--sysconfdir=/etc \
 		--mandir=/usr/share/man \
 		--localstatedir=/var \
+                --enable-freedreno \
```
Comment (jirutka) [inline main/libdrm/APKBUILD]: There are spaces instead of tabs. Please use tabs for indentation (and alignment) in APKBUILDs.

**Pull request closed without merging**